bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/PutAdapter.java

```diff
@@ -31,16 +31,17 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Collection;
 import java.util.List;
 import java.util.Map.Entry;
 
 /**
- * Adapt an HBase Put Operation into an Bigtable RowMutation
- *
+ * Adapt an HBase {@link Put} Operation into a Bigtable {@link MutateRowRequest.Builder} or
+ * {@link MutateRowsRequest.Entry}.
  * @author sduskis
  * @version $Id: $Id
  */
-public class PutAdapter implements OperationAdapter<Put, MutateRowRequest.Builder> {
+public class PutAdapter extends MutationAdapter<Put> {
   private final int maxKeyValueSize;
   private final boolean setClientTimestamp;
   public Clock clock = Clock.SYSTEM;
```

Comment:
Not from this change, but does this have to be public? I imagine it's used for testing but is package protected sufficient?